examples/applications/topics_extraction_with_nmf_lda.py

```diff
@@ -1,6 +1,7 @@
 """
 =======================================================================================
-Topic extraction with Non-negative Matrix Factorization and Latent Dirichlet Allocation
+Topic extraction with Non-negative Matrix Factorization and Latent 
+Dirichlet Allocation
 =======================================================================================
```

Comment:
ah sorry for the oversight on my part, I've corrected this. Thanks for taking the time to review.